doc/release-process.md

```diff
@@ -28,10 +28,10 @@ Release Process
 
 * Update hardcoded [seeds](/contrib/seeds/README.md), see [this pull request](https://github.com/bitcoin/bitcoin/pull/7415) for an example.
 * Update [`src/chainparams.cpp`](/src/chainparams.cpp) m_assumed_blockchain_size and m_assumed_chain_state_size with the current size plus some overhead (see [this](#how-to-calculate-m_assumed_blockchain_size-and-m_assumed_chain_state_size) for information on how to calculate them).
```

Comment:
Note: this link seems non-existant

Reply:
Oh, true, interesting that clicking the link still jumps down to the intended section 👀 
obviously this wouldn't work as soon as the section headline text changes -- I guess using a named anchor would be the better way, apparently that's possible with HTML in markdown: https://stackoverflow.com/questions/5319754/cross-reference-named-anchor-in-markdown/7335259#7335259 (untested)

Reply:
Done (thanks!)